qlib/workflow/__init__.py

```diff
@@ -282,6 +285,12 @@ def get_uri(self):
         """
         return self.exp_manager.uri
 
+    def reset_default_uri(self, uri: Text):
```

Comment:
Same as the one in `expm`.

Reply:
Yeap, to provide an api to reset default uri. Use to query experiments with different uri.